src/main/kotlin/su/arlet/business1/core/enums/ArticleStatus.kt

```diff
@@ -0,0 +1,7 @@
+package su.arlet.business1.core.enums
+
+enum class ArticleStatus {
+    ON_REVIEW,
+    NEED_FIXES,
+    APPROVED,
```

Comment:
PUBLISHED?
may be also DELETED or CANCELED or ARCHIVE

Reply:
PUBLISHED - ok
CANCELED and others - not in the business model